docs/source/conf.py

```diff
@@ -0,0 +1,221 @@
+# -*- coding: utf-8 -*-
+#
+# Configuration file for the Sphinx documentation builder.
+#
+# This file does only contain a selection of the most common options. For a
+# full list see the documentation:
+# http://www.sphinx-doc.org/en/master/config
+
+# -- .md -> .rst setup  ------------------------------------------------------
+import recommonmark
+from recommonmark.parser import CommonMarkParser
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+import sys
+
+sys.path.insert(0, os.path.abspath("../.."))
+
+
+# -- Project information -----------------------------------------------------
+
+project = "Sparkle"
+copyright = (
+    "2020, Andy Cheon, Stephanie Jung, Collin Prather, Matthew Sarmiento, Kevin Wong"
+)
+author = "Andy Cheon, Stephanie Jung, Collin Prather, Matthew Sarmiento, Kevin Wong"
+
+# The short X.Y version
+version = ""
+# The full version, including alpha/beta/rc tags
+release = "0"
+
+
+# -- General configuration ---------------------------------------------------
+
+# If your documentation needs a minimal Sphinx version, state it here.
+#
+# needs_sphinx = '1.0'
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = [
+    "sphinx.ext.autodoc",
```

Comment:
We may want to include `sphinx.ext.napoleon` to enable writing docstrings following Google's python [styleguide](https://google.github.io/styleguide/pyguide.html#38-comments-and-docstrings).